h5p-bildetema-words-topic-image/language/.en.json

```
@@ -102,6 +102,15 @@
           "default": "Pause audio"
         }
       ]
+    },
+    {
+      "label": "Show written words label",
```

Since this is not the label/translation, but the boolean value, the label for it should be "Show written words"
```suggestion
      "label": "Show written words",
```